src/main/java/org/folio/fqm/service/CrossTenantQueryService.java

```diff
@@ -88,26 +61,31 @@ public List<String> getTenantsToQuery(EntityType entityType, boolean forceCrossT
     return tenantsToQuery;
   }
 
-  public String getCentralTenantId() {
+  @SuppressWarnings("unchecked") // JsonPath.parse is returning a plain List without a type parameter, and the TypeRef (vs Class) parameter to JsonPath.read is not supported by the JSON parser
+  private List<Map<String, String>> getUserTenants() {
     try {
-      String rawJson = ecsClient.get(CONSORTIA_CONFIGURATION_PATH, Map.of());
-      DocumentContext parsedJson = JsonPath.parse(rawJson);
-      return parsedJson.read(CENTRAL_TENANT_ID);
+      String userTenantsResponse = ecsClient.get("user-tenants", Map.of());
+      return JsonPath
+          .parse(userTenantsResponse)
+          .read("$.userTenants", List.class);
     } catch (Exception e) {
-      log.debug(CROSS_TENANT_QUERY_ERROR);
-      return null;
+      log.error(CROSS_TENANT_QUERY_ERROR, e);
```

Comment:
nit (feel free to ignore): I'd keep this as debug to avoid cluttering the logs with lots of ERRORs when running queries in non-ECS environments. Since for any non-ECS environment, you'd be seeing this message every time even though there's not really an error running the query.

Reply:
Honestly, I should probably remove the `try` block here, since exceptions would be truly exceptional. We should still get a response in non-ECS envs

Reply:
Oh yeah good point, I was thinking of the old logic